lib/features.js

```diff
@@ -88,6 +88,15 @@ const features = {
                  'capabilities or host volume mounts.',
     defaults: false,
     module: require('./features/dind')
+  },
+
+  interactive: {
+    title: 'Docker Exec Websocket Server',
+    description: 'Runs another process alongside the original executed process' + 
+                 'and pipes the result through websockets. Can be used for SSH-like' +
+                 'access to docker containers.',
```

Comment:
Nit, but this is the description that a task writer sees.

So probably has to be:
This allows you to interactively run commands inside the container and attaches you to the stdin/stdout/stderr over a websocket. Can be used to for SSH-like...